modules/manually-create-identity-access-management.adoc

```diff
@@ -15,22 +15,55 @@ ifeval::["{context}" == "manually-creating-iam-gcp"]
 endif::[]
 
 [id="manually-create-iam_{context}"]
-== Manually create IAM
+= Manually create IAM
 
-The Cloud Credential Operator can be put into manual mode prior to installation
-in environments where the cloud identity and access management (IAM) APIs are
-not reachable, or the administrator prefers not to store an administrator-level
-credential secret in the cluster `kube-system` namespace.
+The Cloud Credential Operator (CCO) can be put into manual mode prior to
+installation in environments where the cloud identity and access management
+(IAM) APIs are not reachable, or the administrator prefers not to store an
+administrator-level credential secret in the cluster `kube-system` namespace.
 
 .Procedure
+ifdef::aws[]
+//credentialsMode=Manual only verified supported on AWS in 4.6 GA
+. Create the `install-config.yaml` file:
++
+[source,terminal]
+----
+$ ./openshift-install create install-config --dir=<installation_directory>
+----
 
+. Edit the `install-config.yaml` configuration file so that the CCO operates in
+manual mode:
++
+[source,terminal]
+----
+$ oc edit cloudcredentials.operator.openshift.io cluster
+----
```

Comment:
That step only shows up for AWS (Azure and GCP use the old way, after generating manifests), does that seem right? Or still not right?

Reply:
@jeana-redhat 
1. for you question:

    > That step only shows up for AWS (Azure and GCP use the old way, after generating manifests), does that seem right? Or still not right?

    right, `credentialsMode: Manual `is only effective AWS. 

2. since at this time the cluster has not been installed yet, so we can not use `oc edit cloudcredentials.operator.openshift.io cluster` command. 
To add `credentialsMode: Manual` option, just need to edit `install-config.yaml` file, e.g. `vi <installation_directory>/install-config.yaml`

Reply:
Thanks for clarifying :)